src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -2654,6 +2776,28 @@ mod tests {
         assert!(test_bed.contains_variable_with_name("PNEU_XBLEED_VALVE_OPEN"));
     }
 
+    #[test]
+    fn wing_anti_ice_simvars() {
+        let test_bed = test_bed();
+
+        assert!(test_bed.contains_variable_with_name("PNEU_WING_ANTI_ICE_SYSTEM_ON"));
+        assert!(test_bed.contains_variable_with_name("PNEU_WING_ANTI_ICE_SYSTEM_SELECTED"));
+        assert!(test_bed.contains_variable_with_name("PNEU_WING_ANTI_ICE_HAS_FAULT"));
+        assert!(test_bed.contains_variable_with_name("PNEU_WING_ANTI_ICE_GROUND_TIMER"));
+        assert!(test_bed.contains_variable_with_name("PNEU_1_WING_ANTI_ICE_CONSUMER_PRESSURE"));
```

Comment:
I think it's better to name the variables like this `PNEU_WING_ANTI_ICE_1_CONSUMER_PRESSURE` or `PNEU_WING_ANTI_ICE_L_CONSUMER_PRESSURE` so that it has the same structure as the other simvars.